session/bootstrap.go

```diff
@@ -617,11 +617,13 @@ const (
 	version89 = 89
 	// version90 converts enable-batch-dml to a sysvar
 	version90 = 90
+	// version92 converts run-auto-analyze to a sysvar
+	version92 = 92
```

Comment:
I assume you picked v92 because [this PR](https://github.com/pingcap/tidb/pull/34644) is 91? 

We can't guarantee the merge order of PRs, so it is best to just pick the next number in sequence and then fix conflicts when they arise.